api/scpca_portal/test/factories.py

```diff
@@ -182,3 +182,25 @@ def add_sample_library_relation(self, create, extracted, **kwargs):
         library = self.libraries.first()
 
         sample.libraries.add(library)
+
+
+class DatasetFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = "scpca_portal.Dataset"
+
+    data = {
+        "SCPCP999990": {
+            "merge_single_cell": False,
+            "includes_bulk": True,
+            Modalities.SINGLE_CELL: ["SCPCS999990", "SCPCS999991"],
+            Modalities.SPATIAL: ["SCPCS999992"],
+        },
+        "SCPCP999991": {
+            "merge_single_cell": True,
+            "includes_bulk": True,
+            Modalities.SINGLE_CELL: ["SCPCS999993", "SCPCS999994"],
+            Modalities.SPATIAL: [],
+        },
+    }
+    email = "dataset@scpca.alexslemonade.org"
```

Comment:
```suggestion
    email = "user@example.org"
```

This is something I have found to be helpful in the long run but when mocking email addresses we should always default to `example.com` as the domain unless otherwise needed.

Reply:
Nice idea